### ISSUE TITLE
cctrial exits with success on ctrl-c

### ISSUE THREAD

**Issue by djmitche** [tardyp/cctrial#1]
I was using cctrial in a `git rebase -i` operation, so that it would automatically go on to the next commit after fixing failures in the current commit.

However, when I hit ctrl-c, instead of halting the rebase with an error, it ran on to the next commit.

**Comment by tardyp** [comment]
yes, the -H option is broken. I have it fixed in one of my computer, but forgot to push it.

**Comment by tardyp** [comment]
but after reading twice your comment, I like your proposed workflow, which is different from what I though. 
You want cctrial -H to re-run until all tests are fixed, and giveup with exit 1 if ctrl C is hit.
